conf/modules.config

```diff
@@ -933,8 +906,28 @@ process{
             }
     }
 
-    //CNVKIT
-    withName: 'CNVKIT_BATCH_SOMATIC' {
```

Comment:
I think we're still missing this one in the `modules.config`.

Reply:
Thanks, that fixed the cnvkit test!